packages/tabs/src/TabTitle.tsx

```diff
@@ -176,9 +173,15 @@ const listTitleStyles = css`
 `;
 
 const listTitleChildrenStyles = css`
-  display: flex;
-  align-items: center;
-  gap: 4px;
+  width: 100%;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
```

Comment:
I don't like that we can't use flexbox here, but text overflow doesn't work the way we need to use it

Reply:
Might be worth adding a comment about why we're not using flex here. Someone (most likely me 🙃) is almost guaranteed to try adding it next time they're in here